pubpub/platform
# Select dropdown

## Issue(s) Resolved

Closes https://github.com/pubpub/platform/issues/613

## High-level Explanation of PR
Renders the select dropdown

## Test Plan

* Add a numeric array or a string array field
* Add to a form as a select dropdown
* Open the form fill page. The select dropdown should render as expected and work to update/create a pub.

## Screenshots (if applicable)

https://github.com/user-attachments/assets/2b9a2c92-4d45-4875-9913-d41bbc16dde6



## Notes
